test/pde_systems/MOL_1D_Integration.jl

```diff
@@ -145,16 +145,16 @@ end
 
     disc = MOLFiniteDifference([x => 120], t)
 
-    prob = discretize(pde_system, disc)
+    @test_throws ArgumentError (prob = discretize(pde_system, disc))
```

Comment:
I don't think this is supposed to throw an error?

Reply:
Moving this conversation to the test PR.